Community PR - Dune Dashboard (2a) (#5976)
w3f/polkadot-wiki: docs/general/dune-analytics/parachain-dashboards/manta-dashboards.md

---
id: manta-dashboards
title: Manta Dashboards
sidebar_label: Manta
description:
  Manta Atlantic is the fastest and most decentralized ZK Layer1 supporting modular on-chain
  compliance identities. Its suite of core products and technologies, including zkNFTs, staking and

>   ecosystem projects, offers user-friendly access to powerful ZK-enabled use cases.

Reviewer: ```suggestion
  Manta Atlantic is a fast and decentralized ZK Layer 1 supporting modular on-chain
  compliance identities.
```

keywords: [polkadot, dashboard, dune, manta]
slug: ../manta-dashboards
---

# Manta Dashboards

## Overview

Manta Atlantic is the fastest and most decentralized ZK Layer1 supporting modular on-chain
compliance identities. Its suite of core products and technologies, including zkNFTs, staking and

> ecosystem projects, offers user-friendly access to powerful ZK-enabled use cases.

Reviewer: ```suggestion
Manta Atlantic is a fast and decentralized ZK Layer 1 supporting modular on-chain
compliance identities. Its suite of core products and technologies, including zkNFTs, staking, and
ecosystem projects, offers user-friendly access to powerful ZK-enabled use cases.
```

## Featured Dashboards on Dune

> Here you'll find a variety of dashboards that help visualize data from the Manta parachain:

Reviewer: ```suggestion
Here you will find a variety of dashboards that help visualize data from the Manta parachain:
```

- [Manta Parachain Staking Dashboard](https://dune.com/substrate/manta): This dashboard provides
  various information on Manta Parachain Staking.

## Key Tables

Data from the manta parachain is organized into several key tables:

- `manta.balances`
- `manta.blocks`
- `manta.calls`
- `manta.events`,
- `manta.extrinsics`
- `manta.transfers`

## Useful Queries

> Currently, there are no specific queries provided. Please check back later for updates.

Reviewer: ```suggestion
Currently, there are no specific queries provided.
```
